modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -567,10 +585,87 @@ module.exports = {
             const right = content.indexOf('>', left);
             const nextLeft = content.indexOf('<', right);
             if ((right !== -1) && (nextLeft !== -1)) {
-              content = content.substr(0, right + 1) + self.apos.util.escapeHtml(doc.title) + content.substr(nextLeft);
+              content = content.substring(0, right + 1) + self.apos.util.escapeHtml(doc.title) + content.substring(nextLeft);
+            }
+          }
+        }
+        return content;
+      },
+      // Quickly replaces inline image placeholder URLs with
+      // actual, SEO-friendly URLs based on `widget._relatedDocs`
+      linkImages(widget, content) {
```

Comment:
Thought about code reuse between this and linkPermalinks, but the URL styles are different enough that I don't want to over-factor things right now